src/chrome/content/rules/Gtimg.cn.xml

```diff
@@ -21,20 +20,19 @@
 				-->
 	<target host="imgcache.gtimg.cn" />
 	
-	<!--	Mixed content
-	<target host="qzonestyle.gtimg.cn" /
+	<!--	Mixed content:Exclusion because of mixed content errors.
+	Detail: https://github.com/EFForg/https-everywhere/pull/4591
 				-->
-
-		<test url="http://imgcache.gtimg.cn/tencentvideo_v1/vstyle/film/v2/img/play/sprite_play.ie8.png" />
-
+	<target host="qzonestyle.gtimg.cn" />
+	<exclusion pattern="^http://qzonestyle\.gtimg\.cn/$" />
```

Comment:
You should remove the `$` otherwise you got still mixed content errors on `http://qzonestyle.gtimg.cn/foo/bar.html`.